xray_test.go

```diff
@@ -703,11 +701,14 @@ func TestCurationAudit(t *testing.T) {
 	output := localXrayCli.RunCliCmdWithOutput(t, "curation-audit", "--format="+string(utils.Json), workingDirsFlag)
 	expectedResp := getCurationExpectedResponse(config)
 	var got []coreCuration.PackageStatus
-	err = json.Unmarshal([]byte(output[strings.Index(output, "["):]), &got)
-	assert.NoError(t, err)
-	assert.Equal(t, expectedResp, got)
-	for k, v := range expectedRequest {
-		assert.Truef(t, v, "didn't recieve expected GET request for packe url %s", k)
+	startIndex := strings.Index(output, "[")
+	if assert.GreaterOrEqual(t, startIndex, 0) {
```

Comment:
I'd prefer to remove the "if" to get a cleaner code. This test is already very long and not so readable. Please consider just using assert.GreaterOrEqual.
If "panic"s disturbed you, you can use require instead of "assert".

Reply:
the issue here is with using output[startIndex:] with -1